app/views/ReportOneLoginProblemPage.scala.html

```diff
@@ -0,0 +1,134 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
```

Comment:
Probably don't need to worry about it, but as this page was made (not updated) this year, does it need to be 2025? 😅

Reply:
Thanks @TimothyFothergill , updated!